templates/pricing/devices.html

```diff
@@ -212,7 +195,7 @@ <h3 class="p-heading--4">Are your devices running Ubuntu Desktop or Server?</h3>
   </div>
 </section>
 
-{% with colour="light", bordered="false" %}{% include "shared/_call-sales.html" %}{% endwith %}
+{% with colour="white", bordered="false" %}{% include "shared/_call-sales.html" %}{% endwith %}
```

Comment:
Making this white removes the visual separation from the previous section, is this intentional?